roles/icinga_agent/defaults/main.yml

```diff
@@ -21,3 +21,7 @@ icinga2_user:
 icinga2_group:
   Debian: nagios
   RedHat: icinga
+
+icinga_feature_folders:
```

Comment:
Why is this a variable? Is it different on diffrent OS?

Reply:
I created this variable simply because the rest is also variablized. But you are right. At this point, it doesn't necessarily have to be a variable.

Reply:
I've changed it to a normal loop and deleted the variable `icinga_feature_folders`.